doc/private_net.rst

```diff
@@ -0,0 +1,101 @@
+      "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" }, "balance": "0x01" },
+      "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" }, "balance": "0x01" },
+      "008a78302c6fe24cc74008c7bdae27b7243a7066" /* <= Enter your account address here */: {
+          "balance" : "0x200000000000000000000000000000000000000000000000000000000000000"
+      }
+    }
+  }
+  
+  
+This includes all forks activated at genesis, all precompiled contracts existing on the main net and initial balance for your account.
+
+In case this format gets out of date, please consult the format of the source files in https://github.com/ethereum/aleth/tree/master/libethashseal/genesis.
+
+**3.** Run aleth with ``config.json``.
+::
+  aleth/aleth --config /Users/johndoe/Documents/config.json -m on -a 008a78302c6fe24cc74008c7bdae27b7243a7066 --no-discovery --pin --unsafe-transactions
+
+``-m on`` enables CPU mining;
+
+``-a 008a78302c6fe24cc74008c7bdae27b7243a7066`` sets the beneficiary of the mined blocks;
+
+``--no-discovery --pin`` effectively disables networking; we have ony single node, we don't need to discover other ones and we don't allow others to connect to us;
+
+``--unsafe-transactions`` disables additional prompt before sending each transaction, we don't need it in the testing environment.
+
+It can take a while to mine the first block, but the mining should go faster further on.
+
+**4.** Meanwhile, open a new window in your CLI, navigate into the ``aleth`` directory, and run
+::
+  scripts/dopple.py
+
+This enables HTTP RPC at http://127.0.0.1:8545 by running a proxy which forwards all HTTP requests to IPC interface of aleth.
+
+Connecting Remix
+------
+
+This assumes that the contract code is already entered and successfully compiled.
+
+**1.** On the ``Run`` tab choose ``Web3 Provider`` in ``Environment`` list.
+
+**2.** After connecting to the node it should fetch your account's address and automatically choose it in the ``Account`` list.
+
+**3.** Use ``Deploy`` button to deploy a compiled contract into the blockchain. Aleth requests the confirmation and the account's password in its CLI - switch to aleth window and confirm the transaction after clicking ``Deploy``
```

Comment:
* Use Deploy -> Use the Deploy
* switch to aleth window -> switch to the aleth window
* confirm the transaction after clicking Deploy -> confirm the transaction when prompted